srcpkgs/wvkbd/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'wvkbd'
+pkgname=wvkbd
+version=0.7
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pango-devel cairo-devel wayland-devel pkg-config"
```

Comment:
the only hostmakedepends should be `wayland-devel` (because `wayland-scanner`) and `pkg-config`

`pango-devel cairo-devel` should be makedepends, and `wayland-devel` should be in both places.